projects/admin/src/app/record/detail-view/document-detail-view/inherited-call-numbers/inherited-call-numbers.component.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2021 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Component, Input, OnInit } from '@angular/core';
+import { ItemHoldingsCallNumberPipe } from '@rero/shared';
+
+@Component({
+  selector: 'admin-inherited-call-numbers',
+  templateUrl: './inherited-call-numbers.component.html'
+})
+export class InheritedCallNumbersComponent implements OnInit {
+
+  @Input() item: any;
+
+  /** Get formatted item call numbers */
+  callNumbers: string;
+
+  /**
+   * Constructor
+   * @param _itemHoldingsCallNumberPipe - ItemHoldingsCallNumberPipe
+   */
+  constructor(
+    private _itemHoldingsCallNumberPipe: ItemHoldingsCallNumberPipe
+  ) { }
+
+  /** OnInit hook */
+  ngOnInit() {
+    this._itemHoldingsCallNumberPipe.transform(this.item).subscribe(result => {
+      this.callNumbers = [result.value, this.item.metadata.second_call_number || null]
+      .filter(element => element != null)
+      .join(' | '); });
+
+  }
+
+}
```

Comment:
For me and myself, this component seems completely overkill. 
I think a Pipe should be more elegant way to do this